server/database/seeds/21_users_languages.js

```diff
@@ -1,13 +1,16 @@
-
 exports.seed = function(knex, Promise) {
   // Deletes ALL existing entries
-  return knex('table_name').del()
-    .then(function () {
+  return knex('users_languages')
+    .del()
+    .then(function() {
       // Inserts seed entries
-      return knex('table_name').insert([
-        {id: 1, colName: 'rowValue1'},
-        {id: 2, colName: 'rowValue2'},
-        {id: 3, colName: 'rowValue3'}
+      return knex('users_languages').insert([
+        { user_id: 1, language_id: 1, language_type: 'primary' },
+        { user_id: 2, language_id: 1, language_type: 'primary' },
+        { user_id: 3, language_id: 2, language_type: 'primary' },
+        { user_id: 4, language_id: 2, language_type: 'primary' },
+        { user_id: 5, language_id: 1, language_type: 'primary' },
+        { user_id: 6, language_id: 2, language_type: 'primary' },
```

Comment:
might want some seeds with language_type = "target"   to track whatever they're trying to learn